etc/config/c++.win32.properties

```diff
@@ -1 +1,38 @@
-supportsBinary=true
+# Default settings for C++ on Windows
+
+compilers=&vc2017:&clang
+postProcess=
+demanglerClassFile=./demangler-win32
+objdumper=objdump
+options=
+supportsBinary=false
+stubRe=\bmain\b
+stubText=int main(void){return 0;/*stub provided by Compiler Explorer*/}
+
+
+# clang compilers
+
+group.clang.compilers=&clang_32:&clang_64
+group.clang.intelAsm=-mllvm --x86-asm-syntax=intel
+group.clang.groupName=clang
+group.clang.compilerType=win32
+group.clang.binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|frame_dummy|\.plt.*)$
```

Comment:
Very much nitpicking, but almost certainly this list of things are wrong for Windows. Maybe leave empty with a note TODO this when (if) we support binary in the same way. For context, this is a list of magic special functions that Linux systems provide that I like to hide from binary disassemblies as they're usually not relevant.

Reply:
Ah, yeah, you're right. Completely forgot to change this once I figured out what it was!

Reply:
For windows, this could contain all the functions internal to the C Runtime (CRT / ucrt) static or even dll ones.